dotnet/src/Experimental/Process.UnitTests/Core/ProcessMapBuilderTests.cs

```diff
@@ -0,0 +1,223 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
wondering how the error messages are dealt with the map step?
same as you get an [] of nullable results, when subscribing to onError you get an [] of errors?

Reply:
yes, I believe so.  if out of 5 steps 3 result in some type of error and 2 don't, three invocations of the error handler seems natural.  i'm hestant to intercept and interpret external failure.